cmake/post_install.cmake

```diff
@@ -17,8 +17,7 @@
 
 if("$ENV{USER}" STREQUAL root)
   foreach(DIR ${CHOWN_DIRS})
-    message(STATUS "Changing ${DIR} ownership to ${OWNER_USER}:${OWNER_GROUP}")
+    message(STATUS "Changing $ENV{DESTDIR}${DIR} ownership to ${OWNER_USER}:${OWNER_GROUP}")
```

Comment:
It shouldn't have `${CMAKE_INSTALL_PREFIX}` as well like the line below it?

Reply:
Actually, the other should look like this one! I missed this picking from internal code.  Thanks!